Framework/Geometry/src/Rasterize.cpp

```diff
@@ -28,16 +28,20 @@ constexpr static V3D Y_AXIS{0, 1, 0};
 constexpr static V3D Z_AXIS{0, 0, 1};
 
 struct CylinderParameters {
+  // cppcheck-suppress-begin unusedStructMember
   double radius;
   double height;
+  // cppcheck-suppress-end unusedStructMember
   V3D centerBottomBase;
   V3D symmetryaxis;
 };
 
 struct HollowCylinderParameters {
+  // cppcheck-suppress-begin unusedStructMember
   double radius;
   double innerRadius;
   double height;
+  // cppcheck-suppress-end unusedStructMember
   V3D centerBottomBase;
   V3D symmetryaxis;
 };
```

Comment:
```suggestion
```

Reply:
I think these can be deleted, there is no reference to them anywhere in the codebase.